generators/kubernetes/files.js

```diff
@@ -118,6 +118,14 @@ function writeFiles() {
             this.template('istio/gateway/grafana-gateway.yml.ejs', 'istio/grafana-gateway.yml');
             this.template('istio/gateway/zipkin-gateway.yml.ejs', 'istio/zipkin-gateway.yml');
             this.template('istio/gateway/kiali-gateway.yml.ejs', 'istio/kiali-gateway.yml');
+        },
+
+        writeKustomize() {
```

Comment:
If i'm not mistaken, this function is never called ! Do you plan to add a question for this ?

Reply:
No, it's part of the default generator. When u run jhipster kubernetes/k8s, kustomization.yml will be generated by default like other manifests. This is an additional option to externalize some configuration so that you can apply changes using either kubectl apply -f or kubectl apply -k (using kustomization.yml)